presets/workspace/models/llama3/model.go

```diff
@@ -93,7 +93,7 @@ type llama3_3_70Binstruct struct{}
 func (*llama3_3_70Binstruct) GetInferenceParameters() *model.PresetParam {
 	return &model.PresetParam{
 		Metadata:                  metadata.MustGet(PresetLlama3_3_70BInstructModel),
-		DiskStorageRequirement:    "140Gi",
+		DiskStorageRequirement:    "220Gi",
```

Comment:
Why 80GB more storage required?

Reply:
This size is the os disk size of the node.
It needs extra disk to place container images.